src/main/java/tutorlink/TutorLink.java

```diff
@@ -34,18 +31,7 @@ public static void main(String[] args) {
 
         Console.printWelcomeMessage();
         while (true) {
-            try {
-                String line = ui.collectUserInput();
-                Command command = parser.parse(line);
-                LOGGER.log(Level.INFO, "Parser created");
-                CommandResult commandResult = command.execute();
-                Console.displayCommandResult(commandResult);
-                if (command.isExit()) {
-                    break;
-                }
-            } catch (MissingNextLineException e) {
-                Console.printMessage(e.getMessage());
-            }
+            //main code goes here
```

Comment:
Prob should put a break here so CI don't get stuck on an infinite loop

Reply:
Got it, thanks.